launcher/src/main/resources/application.properties

```diff
@@ -30,6 +30,9 @@ application.jwt.refresh-token.validity-seconds=360000
 application.axon.retry.interval-milli-seconds=1000
 application.axon.retry.max-count=1
 application.axon.retry.pool-size=1
+application.axon.event-processor.default-group=true
+application.axon.event-processor.type=switching
+application.axon.event-processor.segments=1
```

Comment:
I think we could possibly relax the constraint here to let each aggregate have its own group and multiple segments per processor.

Reply:
The benefits aren't going to be great for the sample app so I don't think it matters much.

But having the properties ready for tweaking in the properties is good to call out their existence.